UDx-container/vsdk-bash

```diff
@@ -114,6 +115,9 @@ HOSTNAME=vsdk-$USER
 
 #         --env LD_LIBRARY_PATH=/opt/vertica/lib 
 
+# this would be an include file, except I don't know how to find
```

Comment:
There are ways to get the directory of the current script.  You could then use that to get the path to some include file.

For instance, this will set the directory where the script is defined:
```
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Reply:
why not simply 

```SCRIPT_DIR="$(dirname \"${BASH_SOURCE[0]}\")"```

?  Does the cd & pwd take care of some problem I'm missing?

Reply:
The cd and pwd handles symlinks.  I stole it from here, which has more of a discussion:
https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itsel